src/ralph/data_center/admin.py

```diff
@@ -132,9 +118,9 @@ class DataCenterAssetAdmin(
     bulk_edit_list = list_display
     search_fields = ['barcode', 'sn', 'hostname', 'invoice_no', 'order_no']
     list_filter = [
-        StatusFilter, BarcodeFilter, SNFilter, HostnameFilter, InvoiceNoFilter,
-        InvoiceDateFilter, OrderNoFilter, ModelFilter, DepreciationDateFilter,
-        ForceDepreciationFilter, RemarksFilter, RackFilter
+        'status', 'barcode', 'sn', 'hostname', 'invoice_no', 'invoice_date',
```

Comment:
what happens if i mis-spell something here? I will get an error during the application startup?